apps/pipelines/tests/test_pipeline_history.py

```
@@ -0,0 +1,530 @@
+    ExperimentSessionFactory,
+)
+from apps.utils.factories.pipelines import PipelineFactory
+from apps.utils.factories.service_provider_factories import LlmProviderFactory
+from apps.utils.langchain import (
+    FakeLlmEcho,
+    build_fake_llm_service,
+)
+from apps.utils.pytest import django_db_with_data
+
+
+@pytest.fixture()
```

Ruff

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)